corehq/apps/export/templates/export/partials/export_list_controller.html

```diff
@@ -44,6 +44,7 @@
                 <pagination data-apply-bindings="false"
                             data-bind="visible: showPagination"
                             params="goToPage: goToPage,
+                                    slug: 'export-list',
```

Comment:
this is fantastic!  Love the small footprint.  What happens if `slug` isn't set?  Just that it won't save the page limit?

Reply:
yep